docs/modules/configuration/pages/jet-configuration.adoc

```diff
@@ -297,6 +297,18 @@ The most important properties are listed here:
 Each job has job-specific configuration options. These are covered
 in detail in xref:pipelines:configuring-jobs.adoc[].
 
+=== Isolated Jobs
+
+To activate isolated jobs, your license key must include `Advanced Compute`.
+
+To isolate computation from storage, you can define the location for Jet jobs. This means that you can manage your workload without worrying that the Jet processing jobs starve resources from your storage components.
+
+NOTE: Your storage components still need to serve the data and this has some impact on their resources. Before using isolated jobs, ensure that the processing element of the job is substanitally more resource-intensive than the data retrieval element. 
+
+Use the `JobBuilder` API to define the default behaviour for your isolated computing. For further information on `JobBuilder`, refer to the link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/jet/JetService.JobBuilder.html[API Reference, window=_blank].
```

Comment:
Users do not need to define a `JobBuilder` API, this is internal and an implementation is already provided. Users simply need to provide a `JetMemberSelector` implementation in the builder, as done here with a provided selector implementation for lite members only: https://github.com/hazelcast/hazelcast-code-samples/blob/master/jet/wordcount-compute-isolation/src/main/java/com/hazelcast/samples/jet/wordcount/WordCountWithComputeIsolation.java#L99

Reply:
Updated